packages/core/content-manager/admin/src/components/Widgets.tsx

```diff
@@ -371,17 +382,26 @@ const ChartEntriesWidget = () => {
       <DonutChartSVG
         data={[
           {
-            label: 'Draft',
+            label: formatMessage({
+              id: 'content-manager.containers.List.draft',
+              defaultMessage: 'Draft',
+            }),
```

Comment:
Is this label displayed on its own anywhere, or is it only shown in the string with the count?